pkg/virt-operator/resource/generate/install/strategy.go

```diff
@@ -218,10 +218,18 @@ func (ins *Strategy) Instancetypes() []*instancetypev1beta1.VirtualMachineCluste
 	return ins.instancetypes
 }
 
+func (ins *Strategy) SetInstancetypes(instancetypes []*instancetypev1beta1.VirtualMachineClusterInstancetype) {
+	ins.instancetypes = instancetypes
+}
+
```

Comment:
I wouldn't want to introduce this just for testing. Can we workaround this differently?

Reply:
I replaced the setters with an interface, PTAL.

Reply:
Thanks, the approach looks good. There are some build issues. I can have a proper look once these are addressed.